.gitignore

```diff
@@ -53,7 +53,7 @@
 
 /spec/fixtures/plugins/my_plugin/auto_generated
 
-/vendor/bundle/*
+/vendor/bundle
```

Comment:
When `/vendor/bundle` was symlinked in CI, this was appearing as a diff in the git index.